lib/qesapdeployment.pm

```diff
@@ -231,14 +232,28 @@ sub qesap_venv_cmd_exec {
     my (%args) = @_;
     croak 'Missing mandatory cmd argument' unless $args{cmd};
     $args{timeout} //= bmwqemu::scale_timeout(90);
-    $args{failok} //= 0;
-    my $ret;
+    croak "Invalid timeout value $args{timeout}" unless $args{timeout};
```

Comment:
I like to be sure `$args{timeout}` is not zero. I think `$args{timeout} //= bmwqemu::scale_timeout(90);` cover the times `qesap_venv_cmd_exec` is called without `timeout` argument at all. In this case value is set to 90 and croak is "useless". I like to cover also the scenario where caller explicitly call `qesap_venv_cmd_exec(timeout => 0)`.
But I can try creating a couple of UT to better figure out if there's a match between what I want and what I coded. :-)

Reply:
In this case, replacing `//=` with `||=` would set the default(90) if:
- it is `undef` (because undef is false)
- it is `false` (0, empty string)

so instead of failing if it's 0 or otherwise false like the croak does, it would set the default 90, without the need for the croak